apps/authentication/signals.py

```diff
@@ -130,6 +137,12 @@ def set_staff_status(sender, instance: GroupMember, created=False, **kwargs):
     user.save()
 
 
+@receiver(post_save, sender=User)
+def set_default_privacy(sender, instance: User, created=False, **kwargs):
```

Comment:
I'm a bit unsure on how signals work, but maybe it should be `OnlineUser`, there is something about them not reacting to subclasses